sensing/pointcloud_preprocessor/src/distortion_corrector/distortion_corrector_node.cpp

```diff
@@ -0,0 +1,119 @@
+// Copyright 2024 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "pointcloud_preprocessor/distortion_corrector/distortion_corrector_node.hpp"
+
+#include "pointcloud_preprocessor/distortion_corrector/distortion_corrector.hpp"
+
+namespace pointcloud_preprocessor
+{
+/** @brief Constructor. */
+DistortionCorrectorComponent::DistortionCorrectorComponent(const rclcpp::NodeOptions & options)
+: Node("distortion_corrector_node", options)
+{
+  // initialize debug tool
+  {
+    using tier4_autoware_utils::DebugPublisher;
+    using tier4_autoware_utils::StopWatch;
+    stop_watch_ptr_ = std::make_unique<StopWatch<std::chrono::milliseconds>>();
+    debug_publisher_ = std::make_unique<DebugPublisher>(this, "distortion_corrector");
+    stop_watch_ptr_->tic("cyclic_time");
+    stop_watch_ptr_->tic("processing_time");
+  }
+
+  // Parameter
+  use_imu_ = declare_parameter("use_imu", true);
```

Comment:
since it is a "new" node, it should include the schema approach

Reply:
Thanks, fixed in https://github.com/autowarefoundation/autoware.universe/pull/7137/commits/4a828867f93aad70a4d6a2c125c161c98dc4f639